modules/dxp/apps/osb/osb-testray/osb-testray-dispatch-task-executor/src/main/java/com/liferay/osb/testray/dispatch/task/executor/internal/dispatch/executor/TestrayDispatchTaskExecutor.java

```diff
@@ -410,36 +411,38 @@ private void _autofill(
 		Map<String, Object> testrayCaseResultPropertiesMap2 =
 			testrayCaseResultObjectEntry2.getProperties();
 
-		List<ObjectEntry> testrayCaseResultsIssues1 =
+		List<ObjectEntry> testrayCaseResultsIssuesObjectEntriesList1 =
 			(List<ObjectEntry>)
 				testrayCaseResultsIssuesObjectEntriesPage1.getItems();
-		List<ObjectEntry> testrayCaseResultsIssues2 =
+		List<ObjectEntry> testrayCaseResultsIssuesObjectEntriesList2 =
 			(List<ObjectEntry>)
 				testrayCaseResultsIssuesObjectEntriesPage2.getItems();
 
 		if (((Long)testrayCaseResultPropertiesMap1.get(
 				"r_userToCaseResults_userId") > 0) &&
-			!testrayCaseResultsIssues1.isEmpty() &&
+			!testrayCaseResultsIssuesObjectEntriesList1.isEmpty() &&
 			((Long)testrayCaseResultPropertiesMap2.get(
 				"r_userToCaseResults_userId") <= 0) &&
-			testrayCaseResultsIssues2.isEmpty()) {
+			testrayCaseResultsIssuesObjectEntriesList2.isEmpty()) {
 
 			recipientTestrayCaseResultObjectEntry =
 				testrayCaseResultObjectEntry2;
 			sourceTestrayCaseResultObjectEntry = testrayCaseResultObjectEntry1;
-			sourceTestrayCaseResultsIssuesItems = testrayCaseResultsIssues1;
+			sourceTestrayCaseResultsIssuesObjectEntriesList =
+				testrayCaseResultsIssuesObjectEntriesList1;
 		}
 		else if (((Long)testrayCaseResultPropertiesMap2.get(
 					"r_userToCaseResults_userId") > 0) &&
-				 !testrayCaseResultsIssues2.isEmpty() &&
+				 !testrayCaseResultsIssuesObjectEntriesList2.isEmpty() &&
```

Comment:
It's because i'm going to create a hash map with Testray Case Id and Testray Case Result, so i thought it might be a good name if i put as testrayCaseIdTestrayCaseResultMap. Do you have a better name idea?